samples/navigation-compose/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+plugins {
```

Comment:
This file missed the migration (probably due to multiple PRs open), hopefully it's fine to fix this in this PR.